ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/event.proto

```diff
@@ -56,6 +57,12 @@ message CreatedEvent {
   // the contract.
   // Required
   repeated string witness_parties = 5;
+
+  // The agreement text of the contract.
+  // We use StringValue to properly reflect optionality on the wire for backwards compatibility.
```

Comment:
```suggestion
  // We use StringValue to properly reflect optionality on the wire for backwards compatibility.
  // This is necessary since the empty string is an acceptable (and in fact the default) agreement
  // text, but also the default string in protobuf.
```